src/DacpacTool/BuildOptions.cs

```diff
@@ -9,6 +9,7 @@ public class BuildOptions : BaseOptions
         public string Name { get; set; }
         public string Version { get; set; }
         public FileInfo Output { get; set; }
+        public FileInfo ProjectPath { get; set; }
```

Comment:
Perhaps rather than passing the path of the project to the tool, we should just explicitly run the tool in the context of the project directory and use the current working directory as our frame of reference?